src/helpers/BasicTable.js

```diff
@@ -0,0 +1,350 @@
+import Table from "@material-ui/core/Table";
```

Comment:
This component is pretty big, can you split this out into subcomponents and perhaps create a subfolder for the subcomponents?